securesystemslib/signer/_sigstore_signer.py

```diff
@@ -66,37 +60,38 @@ def to_dict(self) -> Dict:
 
     def verify_signature(self, signature: Signature, data: bytes) -> None:
         # pylint: disable=import-outside-toplevel,import-error
-        result = None
         try:
-            from sigstore.verify import VerificationMaterials, Verifier
+            from sigstore.errors import VerificationError as SigstoreVerifyError
+            from sigstore.models import Bundle
+            from sigstore.verify import Verifier
             from sigstore.verify.policy import Identity
-            from sigstore_protobuf_specs.dev.sigstore.bundle.v1 import Bundle
+        except ImportError as e:
+            raise VerificationError(IMPORT_ERROR) from e
 
+        try:
             verifier = Verifier.production()
             identity = Identity(
                 identity=self.keyval["identity"], issuer=self.keyval["issuer"]
             )
-            bundle = Bundle().from_dict(signature.unrecognized_fields["bundle"])
-            materials = VerificationMaterials.from_bundle(
-                input_=io.BytesIO(data), bundle=bundle, offline=True
-            )
-            result = verifier.verify(materials, identity)
+            bundle_data = signature.unrecognized_fields["bundle"]
+            bundle = Bundle.from_json(json.dumps(bundle_data))
```

Comment:
It's a shame that they didn't expose the `from_dict` method of the inner `_Bundle`, but I get it.